Dockerfile

```diff
@@ -12,7 +14,8 @@ RUN set -x \
       && mv pkg/linux_amd64/envconsul ../bin
 
 COPY . /go/src/github.com/gocardless/theatre
-RUN make VERSION=$(cat VERSION) build
+
+RUN make -j 3 VERSION=$(cat VERSION) build
```

Comment:
This usually isn't a good idea, as Go parallelises everything internally anyway. I'd recommend removing the -j for that reason.